saleor/plugins/webhook/plugin.py

```diff
@@ -2036,15 +2050,34 @@ def product_variant_stocks_updated(
             )
         return previous_value
 
+    def _get_webhooks_for_checkout_events(
+        self,
+        event_type: str,
+        checkout: "Checkout",
+        webhooks: Iterable["Webhook"] | None = None,
+    ) -> Iterable["Webhook"]:
+        """Get webhooks for checkout events.
+
+        Fetch all valid webhooks and filter out the ones that have a subscription query
+        with channel filter that doesn't match to the checkout.
+        """
+        return self._get_webhooks_for_channel_events(
+            event_type, checkout.channel.slug, webhooks
```

Comment:
question: Wouldn't  `checkout.channel.slug` generate an additional database query for us?

Reply:
To trigger the checkout event, you need to call `call_checkout_info_event`. To use `call_checkout_info_event` you need to have `checkoutInfo`. 
Fetching `checkoutInfo`, calls:
`channel = checkout.channel`
so, it will not make any additional DB queries